webpack.base.js

```diff
@@ -51,10 +51,10 @@ module.exports = ({ entry, mode, filename, datacenter }) => ({
 
   plugins: [
     new BannerPlugin({
-      banner({ filename }) {
-        const env = filename.match(SUFFIX_REGEXP)[1]
-        const newFileName = filename.replace(SUFFIX_REGEXP, '')
-        return `\n${filename} IS DEPRECATED, USE ${newFileName} WITH { site: 'datadoghq.${
+      banner({ filename: fileName }) {
+        const env = fileName.match(SUFFIX_REGEXP)[1]
+        const newFileName = fileName.replace(SUFFIX_REGEXP, '')
+        return `\n${fileName} IS DEPRECATED, USE ${newFileName} WITH { site: 'datadoghq.${
```

Comment:
needed?

Reply:
This was giving error for the `no-shadow` rule, but now it seems to work fine.